migration_data/release_1_2_0.py

```diff
@@ -126,6 +126,18 @@ def get_blocks_data(blocks):
         -- drop aux stuff
         drop table blockstorage_data;
         alter table tx_positions drop constraint aux_tx_positions_unique_constraint cascade;
+
+        -- create indices
+        CREATE INDEX IF NOT EXISTS tx_positions_hash_index
```

Comment:
Good changes I also wanted to propose it, after I saw this:
https://youtu.be/aaecM4wKdhY?t=2390